release/CHANGELOG.txt

```diff
@@ -0,0 +1,5 @@
+# Helsinki Design System, Icon kit, version 2.10.0
```

Comment:
This file could be part if icon folder or it could have some icon-library-template - prefix.

Reply:
I guess the version number has to be configurable or there should be a way to override it

Reply:
Good catch, should be renamed to reflect the fact that it's a template.

Is it possible to get the correct version number or would it need to bump it too? Would be nice to keep it automatically up to date indeed.

Reply:
I think it could be possible to pass version number from React package to changelog text. It could some placeholder here, $version for example. An then it could be replaced in build

Reply:
So the place to fetch next release version would be from React package?

Reply:
I think package.json version could be used in build phase 🤔

Reply:
Version number is now bumped from the react package.json.

Reply:
> This file could be part if icon folder or it could have some icon-library-template - prefix.
Seconded. It seems that these files are related to icon kit so the folder name `release` should relate to that or else it's confusing. Maybe just `icon-kit` named folder?

Reply:
The prefix would make the idea of that file's use more clear.

My initial thought was to name the folder `release` because it could hold all the templates for all kit licenses. So during release, automation could fetch the templates from this folder and copy them to the release kit zip files. All the kit licenses are listed here https://helsinkisolutionoffice.atlassian.net/l/cp/CyMoiAm5

Reply:
Oh okay you had continuation in mind. Then it's good 👍